analyzers/tests/SonarAnalyzer.Test/TestCases/UnusedPrivateMember.cs

```diff
@@ -410,6 +410,15 @@ public void M39()
     }
 }
 
+internal class MyClass
+{
+    protected MyClass() // Compliant
+    {
+        var a = 1;
+    }
+}
```

Comment:
Good test! I actually meant one with a private constructor (should be Noncompliant with a slightly different message).

Reply:
The message will be the same (the default) as the first DataRow in `UnusedPrivateMember_NonPrivateConstructorInPrivateClass`: `Remove the unused private constructor 'blabla'.`
I'll add this as well anyway 👍

Reply:
You are absolutely right, of course. Still a good case to test!